_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -374,33 +374,23 @@ public static function register_endpoints() {
 	}
 
 	public static function get_plans( $request ) {
-		/**
-		 * Filter to turn off caching of Jetpack plans
-		 *
-		 * @since 5.9.0
-		 *
-		 * @param bool true Whether to cache Jetpack plans locally
-		 */
-		$use_cache = apply_filters( 'jetpack_cache_plans', true );
```

Comment:
Can we add this filter to a list of deprecated filters in case someone somewhere uses it? Thanks!